### DISCUSSION TITLE
Get Column and Row Info of Parquet

### DISCUSSION THREAD

**Discussion by hkpeaks** [duckdb/duckdb#7796] [General]
I want to know how to config script or write C++ code to get the following parquet information from DuckDB by given file path of a parquet file

1. Column Name and Its Data Type
2. Number of Row Group
3. Compression Type
4. Get all data from a select row group for all columns
5. Get all data from a select row group for selected columns
Return data: a) csv file b) csv bytestream c) arrow

**Comment by Mause** [comment]
Have you had a look at the parquet metadata functions? https://duckdb.org/docs/data/parquet/metadata

**Comment by hkpeaks** [comment]
I have a review the list include in the meta data.
How to get data from selected columns of particular row group of a parquet file?
